### PR TITLE
Copyright and licensing

### PR DESCRIPTION
**Pull request by euronion**
I added new copyright and licensing information to all files and made
the repository (nearly) [REUSE compliant](https://reuse.software/spec/).

Somehow some minor commits got mixed up in this PR (sorry, i don't want
make the effort of taking them out of this PR).

@coroa Please let me know if you agree with this.

### CODE REVIEW

File: atlite/__init__.py [hunk 1]
```diff
@@ -1,29 +1,13 @@
...
-
-## You should have received a copy of the GNU General Public License
-## along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-
-"""
-Renewable Energy Atlas Lite (Atlite)
-
-Light-weight version of Aarhus RE Atlas for converting weather data to power systems data
-"""
```
Comment (coroa) [inline atlite/__init__.py]: Did you remove the package doc string intentionally? It is what is displayed, when you import the library. And then ask for help for it, ie
```python
import atlite
?atlite
```

Instead I would like to extend it with a link to the documentation and perhaps one or two very simple examples.

File: README.md [hunk 2]
```diff
@@ -50,21 +57,21 @@ to file issues or make pull requests.
...
+Copyright (C) 2016-2019 The Atlite Authors:
+
+* Jonas Hörsch (RLI)
+* Tom Brown (KIT)
+* Johannes Hampp (JLUG)
+* Gorm Andresen (AU)
+* David Schlachtberger (FIAS)
+* Markus Schlott (FIAS)
+
+Follow [this link for details](https://atlite.readthedocs.io/en/latest/contributing.html#contributiors-and-copyrights).
```
[the remark this answers is not in the document]
Comment (coroa) [reply README.md]: Same goes for `contributing.rst`.

**Comment by coroa** [comment]
Thanks for that tedious work! Looks good and i would merge soonish, so that we do not pick up anymore conflicts!

File: atlite/__init__.py (hunk 1 above)
Comment (euronion) [reply atlite/__init__.py]: Yes, multiple reasons:

* Previous one was so out-of-date it was more confusing than helping.
* The docstring was repeated in every section of the API ref. generated by automodule on readthedocs.org - not something beautiful to look at.
* I didn't have a nice substitute.

I'd drop the examples (Pareto principle + we are working on nice examples in parallel). So I argue no need to open up a new frontier there IMHO.
Let's stick to a simple description, suggestion in the updated PR.

**Comment by euronion** [comment]
> Thanks for that tedious work! Looks good and i would merge soonish, so that we do not pick up anymore conflicts!

Happy to hear & you're welcome!
Agreed, let's hopefully merge soon, I have already some more local merge conflicts because of this.

**Comment by euronion** [comment]
I merged upstream into my local and resolved the merge conflicts.
Hopefully I didn't make a mistake out of lacking attention and sleep.

If you're ok with the result, then please push the button. :)

**Pull request merged**